samples/keys/identity_key_usage/README.rst

```diff
@@ -0,0 +1,75 @@
+.. _identity_key_usage:
+
+Identity Key Usage
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use a previously written identity key with the PSA crypto APIs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The identity key is an assymentric key of type secp256r1 which can be used for attestation
+services. The identity key requires the Master Key Encryption Key(MKEK) Hardware Unique Key(HUK)
+to be written in the Key Management Unit(KMU). The sample uses the identity key, which also needs
+to be already populated in the KMU.
+
+The sample performs the following operations:
+
+1. The Platform Security Architecture (PSA) API is initialized.
+#. Checks if the Identity key is written to the KMU.
+#. The identity key is read from the KMU.
+#. The identity key is imported to the PSA crypto keystore.
+#. The public key is exported from the identity key.
+
+
+Configuration
+*************
+
+|config|
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/keys/identity_key_usage`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+|test_sample|
+
+      1. |connect_terminal_specific|
+      #. Reset the kit.
+      #. Observe the following output:
+
+         .. code-block:: console
+
+             Initializing PSA crypto.
+             Reading the identity key.
+             Importing the identity key into PSA crypto.
+             Exporting the public key corresponding to the identity key.
+             Success!
+
+         If an error occurs, the sample prints a message and raises a kernel panic.
+
+   If an error occurs, the sample prints a message and raises a kernel panic.
+
+Dependencies
+************
+
+This sample uses the following libraries:
+
```

Comment:
Isn't having pre-populated keys a dependency?

And how would I populate those? First flash the other sample?

Does the KMU survive between flashes?